src/routes/art/ArtGallery.svelte

```diff
@@ -1,18 +1,16 @@
-<script>
+<script lang="ts">
 	import ArtGalleryItem from './ArtGalleryItem.svelte';
 
-	export let data = [
-		{
-			contributor: 'contributor_0',
-			images: ['/img/art/art-1.png']
-		}
-	];
+	export let data: any[] = [];
```

Comment:
let's type it instead of using `any[]` 

```ts
	import ArtGalleryItem from './ArtGalleryItem.svelte';

	export let data: ArtSubmissionData[] = [];
```